server/events/vcs/gitlab_client.go

```diff
@@ -387,12 +387,12 @@ func (g *GitlabClient) UpdateStatus(repo models.Repo, pull models.PullRequest, s
 			break
 		}
 		if i != retries {
-			g.logger.Debug("Head pipeline not found for merge request %d, source '%s'. Retrying in %s",
-				pull.Num, mr.HeadPipeline.Source, delay)
+			g.logger.Debug("Head pipeline not found for merge request %d. Retrying in %s",
```

Comment:
Is there another var that contains the source repo?

Reply:
https://docs.gitlab.com/ee/api/merge_requests.html#response
from the merge request API this is just the type of event that triggered the pipeline which doesn't seem important to log when there is no pipeline yet